src/test/java/com/ioet/bpm/bpmorganizationsapi/BpmorganizationsapiApplicationTests.java

```diff
@@ -81,8 +86,29 @@ public void listAllOrganizations(){
     }
 
     @Test
-    public void get_response_root_endpoint() {
+    public void testCreateOrganization(){
+        Organization organizationToCreate = mock(Organization.class);
+        Organization organizationCreated = mock(Organization.class);
+        Optional<Organization> organization = Optional.of(Mockito.mock(Organization.class));
+
 
+        when(organizationCreated.getUid()).thenReturn("1");
+        assertEquals("1",organizationCreated.getUid());
+
+        verify(organizationCreated, times(1)).getUid();
     }
 
```

Comment:
These tests don't make sense at all.

Reply:
@enriquezrene all the test or only this one?